### ISSUE TITLE
how to extract model metadata

### ISSUE THREAD

**Issue by Shreeshrii** [mittagessen/kraken#175]
Kraken list only shows one model and I am able to use `kraken get` and `kraken show` with that model ID.

However, I am not able to extract the model metadata from any models that I am training, which I have saved as 
 ~/kraken/deva/models/devanew_save.mlmodel  
 ~/kraken/deva/models/devatrain_best.mlmodel

**Comment by mittagessen** [comment]
What kind of metadata do you mean? When you load the model as a
TorchVGSLModel there's a field called user_metadata which contains the
validation error (and more in the development branch) and a field codec
that has the codec. All other metadata is provided manually during the
upload procedure (`ketos publish`, needs a zenodo API key).

**Comment by Shreeshrii** [comment]
>When you load the model as a
TorchVGSLModel there's a field called user_metadata which contains the
validation error (and more in the development branch) and a field codec
that has the codec.

Is there a way to access that?

How do I get the latest beta version? 
sudo pip install . 
had installed 2.0.8

**Comment by Shreeshrii** [comment]
>What kind of metadata do you mean? 

>All other metadata is provided manually during the upload procedure (`ketos publish`, needs a zenodo API key).

kraken show 10.5281/zenodo.2577813
name: 10.5281/zenodo.2577813

A generalized model for English printed text

This model has been trained on a large corpus of modern printed English text\nau
scripts: Latn
alphabet: !"#$%&'()+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]`abcdefghij
accuracy: 99.95%
license: Apache-2.0
author(s): Kiessling, Benjamin
date: 2019-02-26
_______

I think it will be useful to provide similar data even for those models that have not been published on zenodo. A lot of this information is already available during training. Here is a list of fields that should be possible to do:


Model ID/Name/Filename

Model Type - Recognition/Segmentation


Segmentation Type - Boxes/Baseline

Scripts:

Alphabet/Codec

Number of Lines in Training Set

Number of Lines in Evaluation Set

Accuracy

**Comment by mittagessen** [comment]
> Model ID/Name/Filename
> Model Type - Recognition/Segmentation
> Segmentation Type - Boxes/Baseline
> Alphabet/Codec
> Accuracy

Those are already model properties, at least when trained on the blla
branch.

> Scripts:

That's a bit more tricky. There's no 1-to-1 mapping between ISO 15924
script identifiers (which are sorely lacking by themselves) and Unicode
scripts. It wouldn't be possible to distinguish between Latf/Latg/Latn
and Syrc/Syre/Syrj/Syrn and there are scripts we are interested in that
aren't even in Unicode but have ISO 15924 identifiers.

> Number of Lines in Training Set
> Number of Lines in Evaluation Set

This is a fairly meaningless metric and keeping track of it across
multi-part/transfer learning is a mess.
